lib/ansible/config/base.yml

```diff
@@ -1703,6 +1703,14 @@ PARAMIKO_HOST_KEY_AUTO_ADD:
   ini:
   - {key: host_key_auto_add, section: paramiko_connection}
   type: boolean
+PARAMIKO_GSS_AUTH:
+  name: gss auth
+  default: False
+  description: 'Enable GSS-API Authentication'
+  env: [{name: ANSIBLE_PARAMIKO_GSS_AUTH}]
+  ini:
+  - {key: gss_auth, section: paramiko_connection}
+  type: boolean
```

Comment:
this is not needed, it shoudl only be added to the plugin itself

Reply:
Thanks. I have pushed a change to remove this and the version_added change suggested by @samdoran.